modules/xmpp/Caps.js

```diff
@@ -133,6 +151,28 @@ export default class Caps extends Listenable {
         this.rooms.forEach(room => room.sendPresence());
     }
 
+    /**
+     * Updates the presences in the rooms based on the current values in externalFeatures.
+     * @param {ChatRoom} room the room.
+     * @private
+     */
+    _updateRoomsWithExternalFeatures(room) {
+        if (this.externalFeatures.size === 0) {
+            room.removeFromPresence('features');
+        } else {
+            const children = [];
+
+            this.externalFeatures.forEach(f => {
+                children.push({
+                    'tagName': 'feature',
+                    attributes: { 'var': f }
+                });
+            });
+
+            room.addToPresence('features', { children });
```

Comment:
Just checking: this will not add multiple `features` XML elements, correct?

Reply:
Yes, just one. with the children.